api/cache/projects.js

```diff
@@ -3,7 +3,7 @@ const airTable = require('../helpers/airTable');
 const arraysHelpers = require('../helpers/arrays');
 const axios = require('axios').default;
 const projectsHelpers = require('../helpers/projects');
-
+const videoHelper = require('../services/vimeo');
```

Comment:
Optional tiny thing, but could rename `videoHelper` to `vimeo` or `vimeoService` (here and below) just so it's even clearer (as it's specifically a Vimeo service rather than one of the things in the `/api/helpers` directory)